api/python/provisioner/utils.py

```diff
@@ -60,6 +60,25 @@ def converter_path(value):
     return value if value is None else Path(str(value))
 
 
+def file_scheme_key_converter(value: dict):
```

Comment:
could you rename to `converter_file_scheme_key` to be aligned with existent code?

Reply:
Yes, sure. Fixed that.